gui/mainwindow.cpp

```diff
@@ -1363,11 +1363,18 @@ void MainWindow::toggleFilterToolBar()
 
 void MainWindow::formatAndSetTitle(const QString &text)
 {
+    QString nameWithVersion = QString(tr("Cppcheck %s")).arg(CppCheck::version());
```

Comment:
It feels wrong to translate "format strings".

Reply:
I don't think it's wrong in general, but here it is questionable whether "Cppcheck" needs to be translatable at all (what should it be translated to?).

Also, I notice I have used `%s` while I should have used `%1` here and on the following lines. I'll amend my PR later today.

Reply:
yes.